ReactiveCocoa/Swift/ObjectiveCBridging.swift

```diff
@@ -140,7 +149,12 @@ extension RACCommand {
 			// FIXME: Workaround for <~ being disabled on SignalProducers.
 			|> startWithSignal { signal, disposable in
 				let bindDisposable = enabledProperty <~ signal
-				disposable.addDisposable(bindDisposable)
+				
```

Comment:
This code can change to just use `<~` now. I don't think we should add anything like that.